raiden/tests/integration/cli/test_cli_development.py

```diff
@@ -13,6 +13,9 @@
         "cli_tests_contracts_version", [RAIDEN_CONTRACT_VERSION], scope="module"
     ),
     pytest.mark.parametrize("environment_type", [Environment.DEVELOPMENT], scope="module"),
+    # This is a bit awkward, the default `chain_id` for the `raiden_testchain` and
+    # `local_matrix_servers` fixtures don't align. Therefore we force it to "smoketest" here.
```

Comment:
what does "don't align" mean?

Reply:
The matrix fixture uses the `chain_id` fixture which defaults to `UNIT_CHAIN_ID` (which is `337`) while `raiden_testchain` uses `smoketest`.

Reply:
https://github.com/raiden-network/raiden/issues/6190